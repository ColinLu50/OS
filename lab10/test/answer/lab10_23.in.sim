18888
1536
1406
977
1392
961
