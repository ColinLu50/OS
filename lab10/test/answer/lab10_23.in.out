FCFS
423 -> 358 -> 928 -> 766 -> 967 -> 786 -> 644 -> 785 -> 489 -> 956 -> 730 -> 832 -> 899 -> 847 -> 983 -> 113 -> 700 -> 99 -> 156 -> 974 -> 159 -> 224 -> 817 -> 438 -> 679 -> 555 -> 644 -> 649 -> 90 -> 800 -> 935 -> 705 -> 257 -> 280 -> 177 -> 650 -> 542 -> 7 -> 764 -> 71 -> 512 -> 356 -> 952 -> 166 -> 896 -> 908 -> 504 -> 473 -> 264 -> 338 -> 160 -> 100 -> 505 -> 566 -> 294 -> 229 -> 973 -> 903 -> 684 -> 135
18888
SSTF
423 -> 438 -> 473 -> 489 -> 504 -> 505 -> 512 -> 542 -> 555 -> 566 -> 644 -> 644 -> 649 -> 650 -> 679 -> 684 -> 700 -> 705 -> 730 -> 764 -> 766 -> 785 -> 786 -> 800 -> 817 -> 832 -> 847 -> 896 -> 899 -> 903 -> 908 -> 928 -> 935 -> 952 -> 956 -> 967 -> 973 -> 974 -> 983 -> 358 -> 356 -> 338 -> 294 -> 280 -> 264 -> 257 -> 229 -> 224 -> 177 -> 166 -> 160 -> 159 -> 156 -> 135 -> 113 -> 100 -> 99 -> 90 -> 71 -> 7
1536
SCAN
423 -> 358 -> 356 -> 338 -> 294 -> 280 -> 264 -> 257 -> 229 -> 224 -> 177 -> 166 -> 160 -> 159 -> 156 -> 135 -> 113 -> 100 -> 99 -> 90 -> 71 -> 7 -> 0 -> 438 -> 473 -> 489 -> 504 -> 505 -> 512 -> 542 -> 555 -> 566 -> 644 -> 649 -> 650 -> 679 -> 684 -> 700 -> 705 -> 730 -> 764 -> 766 -> 785 -> 786 -> 800 -> 817 -> 832 -> 847 -> 896 -> 899 -> 903 -> 908 -> 928 -> 935 -> 952 -> 956 -> 967 -> 973 -> 974 -> 983
1406
C-SCAN
423 -> 358 -> 356 -> 338 -> 294 -> 280 -> 264 -> 257 -> 229 -> 224 -> 177 -> 166 -> 160 -> 159 -> 156 -> 135 -> 113 -> 100 -> 99 -> 90 -> 71 -> 7 -> 0 -> 992 -> 983 -> 974 -> 973 -> 967 -> 956 -> 952 -> 935 -> 928 -> 908 -> 903 -> 899 -> 896 -> 847 -> 832 -> 817 -> 800 -> 786 -> 785 -> 766 -> 764 -> 730 -> 705 -> 700 -> 684 -> 679 -> 650 -> 649 -> 644 -> 566 -> 555 -> 542 -> 512 -> 505 -> 504 -> 489 -> 473 -> 438
977
LOOK
423 -> 358 -> 356 -> 338 -> 294 -> 280 -> 264 -> 257 -> 229 -> 224 -> 177 -> 166 -> 160 -> 159 -> 156 -> 135 -> 113 -> 100 -> 99 -> 90 -> 71 -> 7 -> 438 -> 473 -> 489 -> 504 -> 505 -> 512 -> 542 -> 555 -> 566 -> 644 -> 649 -> 650 -> 679 -> 684 -> 700 -> 705 -> 730 -> 764 -> 766 -> 785 -> 786 -> 800 -> 817 -> 832 -> 847 -> 896 -> 899 -> 903 -> 908 -> 928 -> 935 -> 952 -> 956 -> 967 -> 973 -> 974 -> 983
1392
C-LOOK
423 -> 358 -> 356 -> 338 -> 294 -> 280 -> 264 -> 257 -> 229 -> 224 -> 177 -> 166 -> 160 -> 159 -> 156 -> 135 -> 113 -> 100 -> 99 -> 90 -> 71 -> 7 -> 983 -> 974 -> 973 -> 967 -> 956 -> 952 -> 935 -> 928 -> 908 -> 903 -> 899 -> 896 -> 847 -> 832 -> 817 -> 800 -> 786 -> 785 -> 766 -> 764 -> 730 -> 705 -> 700 -> 684 -> 679 -> 650 -> 649 -> 644 -> 566 -> 555 -> 542 -> 512 -> 505 -> 504 -> 489 -> 473 -> 438
961
